### PR TITLE
Cleanup based on pylint

### PR DESCRIPTION
**Pull request by IvarsKarpics**
* Updated gui file (removed references to hwobj in brick where api is used, for example TreeBrick)
* Removed if True: blocks
* Sorted imports
* Removed not used variables
* Converted some come from CamelCase to snake_case

### CODE REVIEW

File: gui/GUISupervisor.py [hunk 1]
```diff
@@ -496,15 +503,15 @@ def finish_init(self, gui_config_file):
...
-                    QtImport.QMessageBox.warning(
-                        self,
-                        "Cannot connect to Hardware Repository",
-                        message,
-                        QtImport.QMessageBox.Retry
-                        | QtImport.QMessageBox.Cancel
-                        | QtImport.QMessageBox.NoButton,
-                    )
-                    == QtImport.QMessageBox.Cancel
+                        QtImport.QMessageBox.warning(
```
Comment (rhfogh) [inline gui/GUISupervisor.py]: Looks like there is an 'eight-spaces' convention behind this. Where does it come from? No objection if there is a standard that says so, but personally I rather prefer the look of the previois version with four spaces less indentation.

File: gui/utils/GUIDisplay.py [hunk 2]
```diff
@@ -741,7 +735,8 @@ def tab_enable_slot(self, enable, page_index=self.indexOf(scroll_area)):
             # add 'tab reset count' slot
             slot_name = "resetTabCount_%s" % label
 
-            def tab_reset_count_slot(self, erase_count, page_index=self.indexOf(scroll_area)):
+            def tab_reset_count_slot(self, erase_count,
```
Comment (rhfogh) [inline gui/utils/GUIDisplay.py]: Are you sure? I tested this in black, and it gave me

            def tab_reset_count_slot(
                self, erase_count, page_index=self.indexOf(scroll_area)
             ):

**Review by rhfogh — CHANGES_REQUESTED** [review]
OK, apart from my couple of comments

File: gui/GUISupervisor.py (hunk 1 above)
Comment (IvarsKarpics) [reply gui/GUISupervisor.py]: Yes indeed it looks strange. It was suggested by running pylint. I will revert back to 4 spaces.

File: gui/utils/GUIDisplay.py (hunk 2 above)
Comment (IvarsKarpics) [reply gui/utils/GUIDisplay.py]: I guess that was my manual fix. I changed it to your suggestion.

**Pull request merged**